client/v2/algod/algod.go

```diff
@@ -53,6 +54,14 @@ func MakeClientWithHeaders(address string, apiToken string, headers []*common.He
 	return
 }
 
+// MakeClientWithTransport is the factory for constructing a Client for a given endpoint with a
+// custom HTTP Transport as well as optional additional user defined headers.
+func MakeClientWithTransport(address string, apiToken string, headers []*common.Header, transport http.RoundTripper) (c *Client, err error) {
```

Comment:
I believe this file is entirely auto-generated, so we'd have to make a modification in the generator repo here: https://github.com/algorand/generator/blob/master/go_templates/client.vm

Reply:
Yeah - I put it in so I could use it cleanly in code pulling this fork.
The weird go-algorand -> sdk[s] relationship I'll leave to you guys.

Reply:
Created https://github.com/TxnLab/go-algorand-sdk/pull/1 for auto-generation

Reply:
sorry - I missed this.  I'll merge.